apps/experiments/tasks.py

```diff
@@ -100,7 +121,7 @@ def get_prompt_builder_response_task(team_id: int, user_id, data_dict: dict) ->
             "id": f"s{int(time.time() * 1000)}",
         }
     )
-    history_event |= {"preview": answer, "time": datetime.now().time().strftime("%H:%M")}
+    history_event |= {"preview": answer, "time": timedelta.now().time().strftime("%H:%M")}
```

Comment:
should this not be `timezone`
```suggestion
    history_event |= {"preview": answer, "time": timezone.now().time().strftime("%H:%M")}
```

Reply:
haha interesting. Nice catch